scripts/release

```diff
@@ -410,12 +410,13 @@ publish_podspec() {
   echo "Verifying Cocoapod trunk permissions"
   pod trunk me > cocoapods_permissions.log
   if grep -Fq "MaterialComponents" cocoapods_permissions.log; then
-    echo "Running a pod trunk push..."
+    echo "Running a pod trunk push from stable..."
+    git checkout stable
```

Comment:
Before pushing, consider checking that this branch is in a clean state:
git status --porcelain

Reply:
we dont do that anywhere else where we switch to the stable branch. (we do that at the beginning of the release though)